src/io/flutter/samples/FlutterSampleManager.java

```diff
@@ -63,4 +78,64 @@
     samples.sort(Comparator.comparing(FlutterSample::getDisplayLabel));
     return samples;
   }
+
+  public static String createSampleProject(@NotNull FlutterSample sample, @NotNull Project project) {
+    final FlutterSdk sdk = FlutterSdk.getFlutterSdk(project);
+    if (sdk == null) {
+      return "unable to find Flutter SDK";
+    }
+
+    final File projectRoot = new File(ProjectUtil.getBaseDir());
+    final String projectNamePrefix = deriveValidPackageName(sample.getElement());
+    final String projectDir = FileUtil.createSequentFileName(projectRoot, projectNamePrefix + "_sample", "");
```

Comment:
I considered it. The problem is we don't know the state of the previous sample -- it could be broken.  Or we might mistakenly identify it as a sample project by name when it is only named that coincidentally.